builder/files/heimdalld.service

```diff
@@ -8,7 +8,7 @@
   WorkingDirectory=/usr/local/bin
   ExecStart=/usr/local/bin/heimdalld start --home /var/lib/heimdall \
```

Comment:
@vcastellm Do we want to change home dir to `$HOME/.heimdalld`, which is currently used in [launch](https://github.com/maticnetwork/launch/blob/master/testnet-v4/sentry/validator/heimdall/setup.sh#L4)?